packages/flutter_tools/lib/src/migrations/swift_package_manager_integration_migration.dart

```diff
@@ -295,9 +385,9 @@ class SwiftPackageManagerIntegrationMigration extends ProjectMigrator {
                   <BuildableReference
                      BuildableIdentifier = "primary"
                      BlueprintIdentifier = "$_runnerNativeTargetIdentifier"
-                     $buildableName
-                     $blueprintName
-                     $referencedContainer
+                     BuildableName = "$buildableName"
```

Comment:
Since the `BlueprintIdentifier` attribute also keeps the XML attribute in the template, I opted for doing that for the others also.